rules_gapic/gapic.bzl

```diff
@@ -56,19 +57,21 @@ gapic_srcjar = rule(
             allow_single_file = True,
             mandatory = True,
         ),
-        "gapic_yaml": attr.label(mandatory = True, allow_single_file = True),
-        "artifact_type": attr.string(mandatory = False, default = "GAPIC_CODE"),
-        "language": attr.string(mandatory = True),
+        "artifact_type": attr.string(mandatory = False), #default = "GAPIC_CODE"
+        "gapic_yaml": attr.label(mandatory = False, allow_single_file = True),
+        "language": attr.string(mandatory = False),
```

Comment:
Why is `language` now optional?

Reply:
Because monolithic generators do not need the language argument, since they generate only one language.

Reply:
microgenerators?

Reply:
Yes, sorry, I meant microgenerators.